en/subscription_services_tos.md

```diff
@@ -0,0 +1,111 @@
+# Mozilla Subscription Services | Terms of Service
+
+Version 1.0, effective October 11, 2022
+{: datetime="2022-10-11" }
+
+Mozilla Corporation offers various services to improve your privacy and security online, including Mozilla VPN and Firefox Relay (the “Services”).
+
+__Mozilla VPN__ is a Virtual Private Network in partnership with [Mullvad](https://mullvad.net). It keeps your internet traffic private from your internet service provider by encrypting and routing it through our partner’s networks instead. It will also replace your actual IP address to keep your personal IP address private from others, such as the websites you visit.
+
+__Firefox Relay__ allows you to keep your primary email address and phone number safe and private by making masks for your email address and phone number that allow you to send and receive emails, calls, and texts without revealing your actual contact information.
+
+Please read these Terms of Service carefully because they explain important information about your use of the Services.
+
+## Subscribing to Mozilla Services
+
+#### You’ll Need A Firefox Account
```

Comment:
Probably don't want to jump from 2nd level to 4th level header? Also in other parts of the doc.

Reply:
fixed.